### DISCUSSION TITLE
Static Site CI - gh-pages commits as username rather than Bot

### DISCUSSION THREAD

**Discussion by casman300** [upptime/upptime#211] [Q&A] [answered]
Thought I posted this in the discussion instead of submitting a bug. As you can see below all commits to the master branch use the upptime-bot. I have looked at the configuration and can see there is a way to change this if required. The interesting bit is that commits to gh-pages show as my username rather than as the bot.

Is there a way to change this at all or is there a limitation of what branches a bot can commit to?

![image](https://user-images.githubusercontent.com/2879023/104486309-78e21c00-55c3-11eb-9cea-0a9f0d1b16ee.png)

Thanks. (upptime is awesome by the way!)

**Comment by AnandChowdhary** [comment]
Thanks! We're discussing this in https://github.com/upptime/upptime/issues/118.

**Comment by AnandChowdhary** [comment, marked as answer]
This is now fixed, we're using [peaceiris/actions-gh-pages](https://github.com/peaceiris/actions-gh-pages) and @upptime respects the committer in your configuration and use it for GitHub Pages.
